core/types/block.go

```diff
@@ -755,3 +771,19 @@ func (b *Block) Hash() common.Hash {
 }
 
 type Blocks []*Block
+
+// currentBlockOntology is used to retrieve the MapContext of a given block.
+func currentBlockOntology(number []*big.Int) []int {
```

Comment:
Unsure if we should have this in the block.go file. Is there a better place we can put this for params?